diff -ruN a/classes/install/Upgrade.inc.php b/classes/install/Upgrade.inc.php
--- a/classes/install/Upgrade.inc.php	2024-08-22 02:51:15.655888100 +0700
+++ b/classes/install/Upgrade.inc.php	2024-08-22 02:51:35.910704300 +0700
@@ -1062,55 +1062,6 @@
 
 			// if it is a remote supp file and article is published, convert it to a remote galley
 			if (!$row['file_id'] && $row['remote_url'] != '' && $article->getStatus() == STATUS_PUBLISHED) {
-				$remoteSuppFileSettingsResult = $submissionFileDao->retrieve('SELECT * FROM article_supp_file_settings WHERE supp_id = ? AND setting_value IS NOT NULL', array($row['supp_id']));
-				$extraRemoteGalleySettings = $remoteSuppFileTitle = array();
-				while (!$remoteSuppFileSettingsResult->EOF) {
-					$rsfRow = $remoteSuppFileSettingsResult->getRowAssoc(false);
-					$remoteSuppFileSettingsResult->MoveNext();
-					switch ($rsfRow['setting_name']) {
-						case 'title':
-							$remoteSuppFileTitle[$rsfRow['locale']] = $rsfRow['setting_value'];
-							break;
-						case 'pub-id::doi':
-						case 'pub-id::other::urn':
-						case 'pub-id::publisher-id':
-						case 'urnSuffix':
-						case 'doiSuffix':
-						case 'datacite::registeredDoi':
-							$extraRemoteGalleySettings[$rsfRow['setting_name']] = $rsfRow['setting_value'];
-							break;
-						default:
-							// other settings are not relevant for remote galleys
-							break;
-					}
-				}
-				$remoteSuppFileSettingsResult->Close();
-
-				// Converted from DAO call to raw SQL because submission_id is no longer available post-schema sync.
-				$articleGalleyDao->update(
-					'INSERT INTO temp_submission_galleys (locale, submission_id, remote_url, label) VALUES (?, ?, ?, ?)',
-					array(
-						$article->getLocale(),
-						$article->getId(),
-						$row['remote_url'],
-						$remoteSuppFileTitle[$article->getLocale()],
-					)
-				);
-				$galleyId = $articleGalleyDao->getInsertId();
-
-				// Preserve extra settings. (Plugins may not be loaded, so other mechanisms might not work.)
-				foreach ($extraRemoteGalleySettings as $name => $value) {
-					$submissionFileDao->update(
-						'INSERT INTO temp_submission_galley_settings (galley_id, setting_name, setting_value, setting_type) VALUES (?, ?, ?, ?)',
-						array(
-							$galleyId,
-							$name,
-							$value,
-							'string'
-						)
-					);
-				}
-
 				// continue with the next supp file
 				continue;
 			}
@@ -1359,7 +1310,7 @@
 			foreach ((array) $submissionFiles as $submissionFile) {
 				$submissionFile->setGenreId($genre->getId());
 				$submissionFile->setUploaderUserId($creatorUserId);
-				$fileStage = $article->getStatus() == STATUS_PUBLISHED ? SUBMISSION_FILE_PROOF : SUBMISSION_FILE_SUBMISSION;
+				$fileStage = SUBMISSION_FILE_SUBMISSION;
 				$submissionFile->setFileStage($fileStage);
 				$submissionFileDao->updateObject($submissionFile);
 			}
@@ -1429,33 +1380,6 @@
 						)
 					);
 				}
-
-				if ($article->getStatus() == STATUS_PUBLISHED) {
-					// Converted from DAO call to raw SQL because submission_id is no longer available post-schema sync.
-					$articleGalleyDao->update(
-						'INSERT INTO temp_submission_galleys (locale, submission_id, file_id, label) VALUES (?, ?, ?, ?)',
-						array(
-							empty($article->getLocale()) ? AppLocale::getPrimaryLocale() : $article->getLocale(),
-							$article->getId(),
-							$submissionFile->getFileId(),
-							$submissionFile->getName(empty($article->getLocale()) ? AppLocale::getPrimaryLocale() : $article->getLocale())
-						)
-					);
-					$galleyId = $articleGalleyDao->getInsertId();
-
-					// Preserve extra settings. (Plugins may not be loaded, so other mechanisms might not work.)
-					foreach ($extraGalleySettings as $name => $value) {
-						$submissionFileDao->update(
-							'INSERT INTO submission_galley_settings (galley_id, setting_name, setting_value, setting_type) VALUES (?, ?, ?, ?)',
-							array(
-								$galleyId,
-								$name,
-								$value,
-								'string'
-							)
-						);
-					}
-				}
 			}
 		}
 		$suppFilesResult->Close();
diff -ruN a/dbscripts/xml/upgrade.xml b/dbscripts/xml/upgrade.xml
--- a/dbscripts/xml/upgrade.xml	2024-08-22 03:42:44.530452000 +0700
+++ b/dbscripts/xml/upgrade.xml	2024-08-22 03:44:13.311787100 +0700
@@ -224,7 +224,6 @@
 	</upgrade>
 
 	<upgrade minversion="3.0.0.0" maxversion="3.1.0.9">
-		<code function="repairSuppFilesFilestage" />
 		<code function="fixAuthorGroup" /><!-- Run again after previous invalid fix (#3289) -->
 	</upgrade>
 
